corehq/ex-submodules/phonelog/models.py

```diff
@@ -4,37 +4,9 @@
 COUCH_UUID_MAX_LEN = 50
 
 
-class OldDeviceReportEntry(models.Model):
-    xform_id = models.CharField(max_length=COUCH_UUID_MAX_LEN)
-    i = models.IntegerField()
-    msg = models.TextField()
-    type = models.CharField(max_length=32)
-    date = models.DateTimeField()
-    server_date = models.DateTimeField(null=True, db_index=True)
-    domain = models.CharField(max_length=100)
-    device_id = models.CharField(max_length=COUCH_UUID_MAX_LEN, null=True)
-    app_version = models.TextField(null=True)
-    username = models.CharField(max_length=100, null=True)
-    user_id = models.CharField(max_length=COUCH_UUID_MAX_LEN, null=True)
-
-    class Meta:
-        db_table = 'phonelog_devicereportentry'
-        app_label = 'phonelog'
-        unique_together = [('xform_id', 'i')]
-        index_together = [
-            ("domain", "date"),
-            ("domain", "device_id"),
-            ("domain", "username"),
-            ("domain", "type"),
-        ]
-
-    def __repr__(self):
-        return u"OldDeviceReportEntry(domain='{}', msg='{}')".format(self.domain, self.msg)
-
-
 @architect.install('partition', type='range', subtype='date', constraint='day', column='server_date')
 class DeviceReportEntry(models.Model):
-    xform_id = models.CharField(max_length=COUCH_UUID_MAX_LEN)
+    xform_id = models.CharField(max_length=COUCH_UUID_MAX_LEN, db_index=True)
```

Comment:
I messed up in my first PR and this index was added even though it is not technically needed. Will drop it in a later PR. (will probably need to do it outside of a migration on ICDS)